### PR TITLE
Fix cosine_decay, l2_normalize, pixel_shuffle API comments

### PR DESCRIPTION
**Pull request by shippingwang**
fix some comments
- [x] pixel_shuffle, fix output shape comments
- [x] l2_normalize, fix 2nd of [PaddlePaddle/FluidDoc#672](https://github.com/PaddlePaddle/FluidDoc/issues/672)
- [x] cosine_decay, fix format

### CODE REVIEW

**Comment by shippingwang** [comment]
预览
image

image

File: python/paddle/fluid/layers/learning_rate_scheduler.py [hunk 1]
```diff
@@ -360,13 +362,13 @@ def cosine_decay(learning_rate, step_each_epoch, epochs):
         Variable: The decayed learning rate.
 
      Examples:
```
Comment (shanyi15) [inline python/paddle/fluid/layers/learning_rate_scheduler.py]: can't find this part in preview pic

Comment (shippingwang) [reply python/paddle/fluid/layers/learning_rate_scheduler.py]: > can't find this part in preview pic

Do we have a better self-check process? for now, the developer will repeat "compile -> preview -> rewrite ->compile", most caused by careless though,  I suggest supporting a tool to preview badly, like just copy these comments to the pasteboard, then preview the comments online without compiling the whole program

**Comment by shippingwang** [comment]
image

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

File: python/paddle/fluid/layers/learning_rate_scheduler.py (hunk 1 above)
Comment (shanyi15) [reply python/paddle/fluid/layers/learning_rate_scheduler.py]: > > can't find this part in preview pic
> 
> Do we have a better self-check process? for now, the developer will repeat "compile -> preview -> rewrite ->compile", most caused by careless though, I suggest supporting a tool to preview badly, like just copy these comments to the pasteboard, then preview the comments online without compiling the whole program

we will consider it, thanks for your suggestion

**Review by shanyi15 — APPROVED** [review]
LG comments

**Pull request merged**